### DISCUSSION TITLE
AX6编译时会自动加上USB驱动，但AX3600不会。

### DISCUSSION THREAD

**Discussion by ericvwv** [Boos4721/openwrt#16] [Q&A/答疑解惑] [answered]
AX6生成的config文件里会默认和安装这些usb驱动，AX3600的config反而不会默认带这几个，也不会安装。
ax6无USB接口，默认安装USB驱动是作何用途啊？

CONFIG_DEFAULT_kmod-usb-dwc3=y
CONFIG_DEFAULT_kmod-usb-dwc3-qcom=y
CONFIG_DEFAULT_kmod-usb3=y
CONFIG_USB_SUPPORT=y

**Comment by Boos4721** [comment, marked as answer]
**DELETED**
